src/backend/AuthService/BusinessLogic/UserSettings/UserSettingsAction.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AuthService.BusinessLogic.Exceptions;
+using AuthService.BusinessLogic.PostLogin;
+using Core.Auth0;
+using Core.Database;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AuthService.BusinessLogic.UserSettings
+{
+    public class UserSettingsAction
```

Comment:
Nie do końca podoba mi się to nazewnictwo. Pamiętaj, że `UserSettingsController`, będzie posiadał wiele akcji. Jak nazwiesz kolejną, `UserSettingsAction2`? Zmień to np na coś precyzyjniejszego: `GetAuth0UsersByCurrentUserAction`

Reply:
Also git